src/main/java/kg/jarkyn/server/Incoming/Requester.java

```diff
@@ -0,0 +1,9 @@
+package kg.jarkyn.server.Incoming;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+
+public interface Requester {
```

Comment:
That name feels a bit too abstract to me. From how I understand it at the moment, that's your wrapped client socket, isn't it?